console/src/components/Services/EventTrigger/Add/AddTrigger.js

```diff
@@ -43,12 +43,13 @@ import {
 class AddTrigger extends Component {
   constructor(props) {
     super(props);
-    this.props.dispatch(loadTableList('public'));
+    this.props.dispatch(loadTableList(props.currentSchema));
```

Comment:
Dispatching from a constructor is a bit unsafe, can we move this to lifecycle methods?

Reply:
This was there in the code so far, 
🙈Do as you wish.

Reply:
It was changed in Scheduled Triggers PR.